tests/test_schema.yaml

```diff
@@ -1284,6 +1292,7 @@ schemas:
         - string
         - "null"
         unique: false
+        sql: varchar(255) null
```

Comment:
without this the tests fail with a foreign key reference error

https://circleci.com/gh/cloudwan/gohan/3081

Reply:
I am now trying something on `sql-remove-circleCI-changes` branch from your branch but I found something weird. It looks as long as we use the same branch, DB is not cleared so it results in unstable testing. I am now trying to find such an information, but it looks some docker container is cached for performance.

Actually, our test will do `drop database` before testing https://github.com/cloudwan/gohan/blob/circleCI-changes/run_test.sh#L8 , but it will not be executed in circleCI because we do not have mysql-client on circle CI. This commit https://github.com/cloudwan/gohan/commit/e7566bd31f3f0a97b9833260fede527f75af1514 could such a fix so can you try that? Thanks.